src/Controls/tests/Xaml.UnitTests/Issues/Issue20438.xaml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Microsoft.Maui.Controls.Xaml.UnitTests.Issue20438"
+             xmlns:local="clr-namespace:Microsoft.Maui.Controls.Xaml.UnitTests"
+             x:Name="page">
+    <ContentPage.BindingContext>
+        <local:Issue20438ViewModel/>
+    </ContentPage.BindingContext>
+    <StackLayout>
```

Comment:
decorate with the correct x:DataType
```suggestion
    <StackLayout x:DataType="local: Issue20438ViewModel">
```

Reply:
@StephaneDelcroix I've removed the duplication as suggested.